docs/tutorials-samples/vs-golden-path.md

```diff
@@ -0,0 +1,105 @@
+
+Prerequisites
+-------------
+
+These scripts assume that .NET Core CLI, Visual Studio Update 2, and Visual Studio tooling are installed on the machine.
+
+A solution using only .NET Core projects
+----------------------------------------
+
+### Writing the library
+
+1. File -> New. Under Templates / .NET Core, choose "Class Library (.NET Core)" and name the project "Library" and the solution "Golden".
+2. Under References, manage NuGet packages, use "nuget.org" as the package source, and browse (including prerelease) for `Newtonsoft.Json`. Install.
+3. Go back to packages and install `Microsoft.NETCore.Portable.Compatibility` from the Core MyGet package source (use an rc2 version). Restore packages. (*)
+4. Open `project.json` and under frameworks/netstandard1.5, add `"imports": ["portable-net40+sl5+win8+wp8+wpa81", "dotnet"]`. Restore packages. (*)
+5. Rename the class "Thing". Add a method: `public int Get(int number) => Newtonsoft.Json.JsonConvert.DeserializeObject<int>($"{number}");`
+6. Verify that the solution builds.
+
+### Writing the test project
+
+1. Create a "test" solution folder. This is only a solution folder, not a physical folder.
+2. Right-click the new "test" solution folder, add/new .NET Core class library project, call it "TestLibrary" and explicitly put it under `Golden/test`.
+3. Right-click References and choose "Add reference". Check "Library" under Projects/solution.
+4. Add references to `Newtonsoft.Json`, `Microsoft.NETCore.Portable.Compatibility`, and add `"imports": [ "portable-net40+sl5+win8+wp8+wpa81", "dotnet" ]` to `frameworks/netstandard1.5` in `project.json`. Restore packages. (*)
+5. Add the `xUnit` and `xunit.runner.visualstudio` packages from nuget.org to the project. Use "portable-net45+win8+wp8+wpa81" in the imports in project.json.
+6. Go back to packages and install `Microsoft.NETCore.Portable.Compatibility` from the Core MyGet package source (use an rc2 version). Restore packages. (*)
+7. Rename `Class1.cs` to `LibraryTests.cs`, add the following code to the class:
+    ```csharp
+        [Fact]
+        public void ThingGetsObjectValFromNumber() {
+            Assert.Equal(42, new Thing().Get(42));
+        }
+    ```
+8. Build. Test -> Windows -> Test Explorer. Run all tests. (this doesn't work)
+
+### Writing the console app
+
+1. Right-click the src folder, and add a new Core console app project, call it App.
+2. Add reference to Projects/Solution/Library.
+3. Add references to `Newtonsoft.Json` (*)
+3.1. Add a dependency to `"System.Runtime.Serialization.Primitives": "4.1.1-rc2-24008"`, references to `Microsoft.NETCore.Portable.Compatibility`, and add `"imports": [ "portable-net40+sl5+win8+wp8+wpa81", "dotnet" ]` to frameworks/netcoreapp1.0 in `project.json`. Restore packages. (*)
```

Comment:
In the above, the asterisks in the two `(*)` are rendered as start/end of italics in Github-flavored Markdown. This can be fixed by escaping the asterisks: `(\*)`.

And `3.1.` is not considered a list item. It should probably be on  a new line inside `3.`:

```md
3. Add references to `Newtonsoft.Json` (\*)

 Add a dependency to `"System.Runtime.Serialization.Primitives": "4.1.1-rc2-24008"`, references to `Microsoft.NETCore.Portable.Compatibility`, and add `"imports": [ "portable-net40+sl5+win8+wp8+wpa81", "dotnet" ]` to frameworks/netcoreapp1.0 in `project.json`. Restore packages. (\*)
```